lms/static/sass/elements/_course-card.scss

```diff
@@ -1,8 +1,46 @@
-.header-img{
-  @include float(left);
-  margin-right:20px;
-}
 .course-card{
-  height: 250px;
-  margin-top:20px;
+  width: 240px;
+  @include susy-media($bp-screen-sm) { width: 240px; }
+  @include susy-media($bp-screen-md) { width: 700px; }
+  @include susy-media($bp-screen-lg) { width: 920px; }
```

Comment:
@schenedx https://github.com/edx/edx-platform/blob/4a9e7c748c668f78f1338e58924bfa3a195a883a/lms/static/sass/views/_program-list.scss#L6
It assumes 12 columns in total.